src/raftstore/store/peer.rs

```diff
@@ -350,6 +355,55 @@ impl Peer {
         Ok(())
     }
 
+    pub fn step(&mut self, m: eraftpb::Message) -> Result<()> {
+        match m.get_msg_type() {
+            eraftpb::MessageType::MsgAppend |
+            eraftpb::MessageType::MsgAppendResponse |
+            eraftpb::MessageType::MsgHeartbeat |
+            eraftpb::MessageType::MsgHeartbeatResponse |
+            eraftpb::MessageType::MsgRequestVote |
```

Comment:
if two peers are isolated, they can send vote to each other. 
So here we can only care Append And Heartbeat.